packages/teuchos/comm/src/Teuchos_StackedTimer.cpp

```diff
@@ -566,6 +604,56 @@ StackedTimer::printLevel (std::string prefix, int print_level, std::ostream &os,
   return total_time;
 }
 
+double
+StackedTimer::printLevelXML (std::string prefix, int print_level, std::ostream &os, std::vector<bool> &printed, double parent_time)
+{
+  // NOTE: If you change the outputting format or logic in this
+  // function, you must make a corresponding change to the function
+  // computeColumnWidthsForAlignment() or the alignments will be
+  // incorrect if the user requests aligned output!
```

Comment:
This is the third place where we replicate this fragile logic. All three functions should change this comment to list all functions that need to be changed if any one of them change.

Reply:
@rppawlo Hmm, I think I should have deleted this comment (obviously, I just copy-pasted printLevel()). printLevelXML() indents according to the current level, but it doesn't support aligning columns.

Reply:
ok. Sounds good @brian-kelley !

Reply:
@rppawlo I had to fix something else, so I deleted this comment.